### PR TITLE
[Bug 1695333] Add script for exporting experiment monitoring data

### PR DESCRIPTION
**Pull request by scholtzan**
This moves the script for exporting experiments monitoring data [from bigquery-etl](https://github.com/mozilla/bigquery-etl/blob/master/script/experiments/export_experiment_monitoring_data.py) to this repo.

The code is pretty much identical to the [existing script in bigquery-etl](https://github.com/mozilla/bigquery-etl/blob/master/script/experiments/export_experiment_monitoring_data.py), with on difference: before it was assumed that datasets to be exported are tables, with the new changes in https://github.com/mozilla/bigquery-etl/pull/1856 datasets can also be views. So when exporting, data is written to a temporary table that can then get exported to GCS. 

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is
  referenced, the pull request should include the bug number in the title)
- [ ] Scan the PR and verify that no changes (particularly to
  `.circleci/config.yml`) will cause environment variables (particularly
  credentials) to be exposed in test logs
- [ ] Ensure the container image will be using permissions granted to
  [telemetry-airflow](https://github.com/mozilla/telemetry-airflow/)
  responsibly.

### CODE REVIEW

File: jobs/experiments-monitoring-data-export/experiments_monitoring_data_export/export.py [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+            `time`,
+            branch,
+            experiment,
+            value
+        FROM {view}
+    """
+
+    query_job = client.query(sql, job_config=job_config)
+    query_job.result()
+    return tmp_table
```
Comment (scholtzan) [inline jobs/experiments-monitoring-data-export/experiments_monitoring_data_export/export.py:92]: This is the main difference compared to the existing export script in bigquery-etl

File: jobs/experiments-monitoring-data-export/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Experiments Monitoring Data Export
+
+This script exports views related to experiment monitoring to GCS as JSON. The script are scheduled
+via Airflow and are executed every 5 minutes to ensure exported data stays up-to-date.
+
+The exported monitoring data is dsiplayed in the Experimenter console and allows experiment owners
```
Comment (jklukas) [inline jobs/experiments-monitoring-data-export/README.md:6]:
```suggestion
The exported monitoring data is displayed in the Experimenter console and allows experiment owners
```

File: jobs/experiments-monitoring-data-export/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,26 @@
+FROM python:3.8
+MAINTAINER REPLACE ME <ascholtz@mozilla.com>
```
Comment (jklukas) [inline jobs/experiments-monitoring-data-export/Dockerfile:2]: This REPLACE ME should probably go away.

**Review by jklukas — APPROVED** [review]
Looks good to me.

**Pull request merged**